charts/pravega/templates/tests/clusterrole.yaml

```diff
@@ -0,0 +1,23 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: "{{.Release.Name}}-pravega-test"
+rules:
+- apiGroups:
+  - pravega.pravega.io
+  resources:
+  - "*"
+  verbs:
+  - "*"
```

Comment:
Does the test pod need to have all access? It seems the condition check is a read-only operation.

Reply:
Fixed this